File: POS System - Retaurant - 2022/Backup/POS System/Maintenance.vb
Public Class Maintenance
    Private Sub cmdUsers_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cmdUsers.Click
        SysUser.MdiParent = Main.Parent
        SysUser.ShowDialog(Me)
    End Sub

    Private Sub cmdBackup_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cmdBackup.Click
        BackupData.MdiParent = Main.Parent
        BackupData.ShowDialog(Me)
    End Sub

    Private Sub cmdUserRole_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles cmdWaiters.Click
        Waiters.MdiParent = Main.Parent
        Waiters.ShowDialog(Me)
    End Sub
End Class

File: POS System - Retaurant - 2022/POS System/Maintenance.cs
using System;
using System.Windows.Forms;

namespace POS_System
{
    public partial class Maintenance
    {
        public Maintenance()
        {
            InitializeComponent();
        }
        private void cmdUsers_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.SysUser.MdiParent = (Form)My.MyProject.Forms.Main.Parent;
            My.MyProject.Forms.SysUser.ShowDialog(this);
        }

        private void cmdBackup_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.BackupData.MdiParent = (Form)My.MyProject.Forms.Main.Parent;
            My.MyProject.Forms.BackupData.ShowDialog(this);
        }

        private void cmdUserRole_Click(object sender, EventArgs e)
        {
            My.MyProject.Forms.Waiters.MdiParent = (Form)My.MyProject.Forms.Main.Parent;
            My.MyProject.Forms.Waiters.ShowDialog(this);
        }
    }
}